src/datachain/lib/file.py

```diff
@@ -489,5 +540,11 @@ def get_file_type(type_: FileType = "binary") -> type[File]:
         file = TextFile
     elif type_ == "image":
         file = ImageFile  # type: ignore[assignment]
+    elif type_ == "video":
+        file = VideoFile
+    elif type_ == "video_clip":
+        file = VideoClip
+    elif type_ == "video_frame":
```

Comment:
how will it initialize frame details?

in general we distinguish files types AFAIR only because they have different behavior for open / read, and other methods ... probably they don't have any additional info

e.g. for VideoClip we can make read / open return us the clip (instead of using an external helper or DataChain uitils)

Reply:
Yes, you're right. Let me think 🤔